ecosystem/typescript/sdk_v2/src/api/faucet.ts

```diff
@@ -0,0 +1,28 @@
+import { fundAccount } from "../internal/faucet";
```

Comment:
We should add support for using auth tokens. Rust client for reference: https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos-rest-client/src/faucet.rs#L53.

Reply:
I believe it's already supported via the aptosConfig through the constructor?  is there a need to add a withAuthToken method?

Reply:
Yes thats true. Can set a Token on AptosConfig. That token would actually go with all requests (fullnode, indexer, faucet) though - I wonder if that could cause issues. @heliuchuan might worth to test that out